resources/docs/custom-rule.md

```diff
@@ -41,8 +41,13 @@ final class MyFirstRector extends AbstractRector
      */
     public function refactor(Node $node): ?Node
     {
+        $nodeName = $this->getName($node->name);
+        if ($nodeName === null) {
+            return null;
+        }
+
         // we only care about "set*" method names
-        if (! $this->isName($node->name, 'set*')) {
+        if (! fnmatch('set*', $nodeName, FNM_NOESCAPE)) {
```

Comment:
Here str_starts_with() would be better choice 👍 (I'm on phone now, so lazy to send PR)

Reply:
Ok, I updated at PR:

- https://github.com/rectorphp/getrector-com/pull/1658